web/src/app/components/loi-editor/loi-editor.component.html

```diff
@@ -19,35 +19,35 @@
     <div class="loi-editor-card-row">
       <div class="loi-editor-card-row-content">
         <div>Predefined sample sites</div>
-  
+
         <div>Import areas or points of interest for which data should be collected.</div>
       </div>
 
       <div
-        *ngIf="canImport"
+        *ngIf="canImport && lois"
         class="loi-editor-card-actions"
       >
         <button *ngIf="lois.size == 0"
           mat-flat-button
           class="import-lois-button"
           (click)="importLois()"
         >
-          <mat-icon class="import-lois-icon">input</mat-icon>
+          <mat-icon class="material-symbols-outlined">input</mat-icon>
           Import
         </button>
 
         <button *ngIf="lois.size > 0"
-          mat-button
+          mat-flat-button
           class="clear-all-lois"
-          (click)="clearLois(survey.id, lois)"
-          color="primary"
+          (click)="clearLois()"
         >
+          <mat-icon class="material-symbols-outlined">delete</mat-icon>
```

Comment:
Can we also change this label to "Delete all" as suggested by @amegantz in https://github.com/google/ground-platform/issues/1570#issuecomment-1986547600 ?